### PR TITLE
[approximate-number] Add types for `approximate-number`

### PR DESCRIPTION
**Pull request by giokaxo**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:

- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/array-bounds/tsconfig.json
Comment (codershiba) [inline types/array-bounds/tsconfig.json:1]: Why is this file being touched?

Comment (giokaxo) [reply types/array-bounds/tsconfig.json:1]: There was merge conflict and editor formatted it so. Fixed in 5eced89c0f61150ba8b16fb7ce81230a8eaeed20

File: types/approximate-number/tslint.json
Comment (codershiba) [inline types/approximate-number/tslint.json:1]: This is also not needed unless you want to disable any eslint rules. Also use `.eslintrc.json` for that purpose.

Comment (giokaxo) [reply types/approximate-number/tslint.json:1]: Updated, thanks 🙏

File: types/approximate-number/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "compilerOptions": {
+        "module": "Node16",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
```
Comment (codershiba) [inline types/approximate-number/tsconfig.json:11]:
```suggestion
```

Comment (codershiba) [reply types/approximate-number/tsconfig.json:11]: Remove `typeRoots` too

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/approximate-number/tsconfig.json
1. types/approximate-number/approximate-number-tests.ts
1. types/approximate-number/package.json
1. types/approximate-number/index.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 0e96ef4ba34af8f0a566e3e778609c0cea1eb00d
</p>

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by jakebailey — APPROVED** [review]

**Pull request merged**